From c1dcd4dfebe6dc92349ed06a3c1b4e8753b0bd50 Mon Sep 17 00:00:00 2001
From: Yannick Fertre <yannick.fertre@foss.st.com>
Date: Thu, 23 Jan 2025 10:20:16 +0100
Subject: [PATCH] arm64: defconfig: switch display drivers from built-in to
 module

Add several drivers from built-in to modudle excepet the driver Simple Drm
Which is necessary to keep alive pixel clock during the boot.

Change-Id: I0599831bb4f093f88b6674cfeb9524bde823b1e7
Signed-off-by: Yannick Fertre <yannick.fertre@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/431952
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
---
 .../fragment-02-defconfig-addons.config       | 22 +++++++++----------
 1 file changed, 11 insertions(+), 11 deletions(-)

--- a/arch/arm64/configs/fragment-02-defconfig-addons.config
+++ b/arch/arm64/configs/fragment-02-defconfig-addons.config
@@ -1,6 +1,6 @@
 #  ALPHABETICALLY SORTED: $LC_ALL=C sort fragment-02-defconfig-addons.config -o fragment-02-defconfig-addons.config
 CONFIG_BACKLIGHT_CLASS_DEVICE=y
-CONFIG_BACKLIGHT_GPIO=y
+CONFIG_BACKLIGHT_GPIO=m
 CONFIG_BT_LE=y
 CONFIG_CLKSRC_STM32_LP=y
 CONFIG_CORESIGHT=y
@@ -18,18 +18,18 @@ CONFIG_CRYPTO_DEV_STM32_CRYP=m
 CONFIG_CRYPTO_DEV_STM32_HASH=m
 CONFIG_DRM=y
 CONFIG_DRM_BRIDGE=y
-CONFIG_DRM_DISPLAY_CONNECTOR=y
-CONFIG_DRM_DW_MIPI_DSI=y
-CONFIG_DRM_I2C_ADV7511=y
+CONFIG_DRM_DISPLAY_CONNECTOR=m
+CONFIG_DRM_DW_MIPI_DSI=m
+CONFIG_DRM_I2C_ADV7511=m
 CONFIG_DRM_KMS_HELPER=y
 CONFIG_DRM_MIPI_DSI=y
 CONFIG_DRM_PANEL=y
 CONFIG_DRM_PANEL_BRIDGE=y
-CONFIG_DRM_PANEL_LVDS=y
-CONFIG_DRM_PANEL_RAYDIUM_RM68200=y
-CONFIG_DRM_PANEL_SIMPLE=y
-CONFIG_DRM_SIMPLEDRM=m
-CONFIG_DRM_STM=m
+CONFIG_DRM_PANEL_LVDS=m
+CONFIG_DRM_PANEL_RAYDIUM_RM68200=m
+CONFIG_DRM_PANEL_SIMPLE=m
+CONFIG_DRM_SIMPLEDRM=y
+CONFIG_DRM_STM=y
 CONFIG_DRM_STM_DSI=m
 CONFIG_DRM_STM_LVDS=m
 CONFIG_DWMAC_GENERIC=y
@@ -117,8 +117,8 @@ CONFIG_STMMAC_ETH=m
 CONFIG_STMMAC_PLATFORM=m
 CONFIG_SYSCON_REBOOT_MODE=m
 CONFIG_TEE_REMOTEPROC=y
-CONFIG_TOUCHSCREEN_GOODIX=y
-CONFIG_TOUCHSCREEN_ILI210X=y
+CONFIG_TOUCHSCREEN_GOODIX=m
+CONFIG_TOUCHSCREEN_ILI210X=m
 CONFIG_TYPEC_UCSI=m
 CONFIG_UBIFS_FS=y
 CONFIG_UCSI_STM32G0=m
